plugins/by-name/none-ls/prettierd.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, config, ... }:
+let
+  cfg = config.plugins.none-ls.sources.formatting.prettierd;
+  ts-ls-cfg = config.plugins.lsp.servers.ts_ls;
+in
+{
+  options.plugins.none-ls.sources.formatting.prettierd = {
+    disableTsServerFormatter = lib.mkOption {
```

Comment:
I'm not a fan of negatively-named booleans. I think it'd be better to have a `enable` option instead of a `disable` option.

It's fine to have a default-true enable option instead of a default-false disable option, for example :slightly_smiling_face:

Reply:
Since this is a copy of the implementation for `prettier`, I kept the same naming scheme. We could diverge for this plugin, but I would suggest keeping the naming the consistent, since `prettierd` is "basically the same".

Reply:
My preference would be to diverge here. Ideally, poor decisions in the past shouldn't influence decisions made today :upside_down_face: 

In separate PRs we could add rename (and/or alias) options for any existing negative booleans.

I don't feel too strongly though. As they're related I'd also be OK with being consistent, it just means there will be _more_ to deprecate/rename if we ever get around to fixing the other option names...

Reply:
I'm sorry, I can't come up with a proper default-true name.

`enableTsServerFormatter` seems confusing, since this option only makes sense if the plugin `ts_ls` is also enabled, which IMO is not the base case.

I'd argue for keeping `disableTsServerFormatter`, but mostly since I can't come up with anything different :D

Reply:
Sure thing. I won't block on a minor naming issue that will need addressing _anyway_ for the other related option.

I'll give other maintainers chance to chime in too, in case they have any opinions or ideas.

Reply:
If we really wanted to not have a negative name, we could do `overrideTsServerFormatter`. But, I'm fine keeping it as-is until we migrate none-ls to new standards.